identity-account/src/account/account.rs

```diff
@@ -212,8 +241,14 @@ impl Account {
   }
 
   /// Push all unpublished changes to the tangle in a single message.
-  pub async fn publish_updates(&mut self) -> Result<()> {
-    self.publish(true).await?;
+  pub async fn publish(&mut self) -> Result<()> {
+    self.publish_internal(true, PublishOptions::default()).await?;
+
+    Ok(())
+  }
+
+  pub async fn publish_with_options(&mut self, options: PublishOptions) -> Result<()> {
+    self.publish_internal(true, options).await?;
```

Comment:
Doc comment public function.

```suggestion
  /// Push all unpublished changes to the Tangle in a single message, optionally choosing
  /// the signing key used or forcing an integration chain update.
  ///
  /// See [`PublishOptions`].
  pub async fn publish_with_options(&mut self, options: PublishOptions) -> Result<()> {
    self.publish_internal(true, options).await?;
```